jabgui/src/main/java/org/jabref/gui/edit/CopyMoreAction.java

```diff
@@ -264,4 +274,70 @@ private void copyKeyAndLink() {
                     Long.toString(entries.size() - entriesWithKey.size()), Integer.toString(entries.size())));
         }
     }
+
+    private void copyField(StandardField field, String fieldDisplayName) {
+        List<BibEntry> selectedBibEntries = stateManager.getSelectedEntries();
+
+        List<String> fieldValues = selectedBibEntries.stream()
+                                                     .filter(bibEntry -> bibEntry.getField(field).isPresent())
```

Comment:
You need to du `getFieldOrAlias` IMHO.

Did you check for date handling? You know, there is `year`, `month`, `day`, and also `date`. If only `year` is present, what does `date` copy do? (It should copy the year -and I hope that `getFieldOrAlias` supports that)